feature/auth/src/main/java/com/ku_stacks/ku_ring/auth/compose/reset_password/ResetPasswordViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.ku_stacks.ku_ring.auth.compose.reset_password
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class ResetPasswordViewModel @Inject constructor() : ViewModel() {
+    private val _sideEffect = Channel<ResetPasswordSideEffect>()
+    val sideEffect = _sideEffect.receiveAsFlow()
+
+    var email by mutableStateOf("")
+    var codeInputFieldEnable = MutableStateFlow(false)
+        private set
```

Comment:
MutableStateFlow를 적용했으면 private set을 거는게 의미가 없습니다.

```suggestion
    private val codeInputFieldEnable = MutableStateFlow(false)
```

Reply:
수정: [f65fd94](https://github.com/ku-ring/KU-Ring-Android/pull/459/commits/f65fd941ccd9eab60efdbac9364bbd333c4abde1)
이 부분은 MutableState로 수정했습니다!